hack/upgrade-test-index-image.sh

```diff
@@ -320,6 +320,13 @@ WH_POD=$( ${CMD} get -n ${HCO_NAMESPACE} pods -l "name=hyperconverged-cluster-we
 ${CMD} logs -n ${HCO_NAMESPACE} "${WH_POD}"
 
 Msg "Brutally delete HCO removing the namespace where it's running"
+# TODO: workaround for https://bugzilla.redhat.com/show_bug.cgi?id=2031919
+# remove once fixed
+if [[ "${INFRASTRUCTURETOPOLOGY}" == "SingleReplica" ]]; then
+  echo "Explictly deleting kubevirt apiservice leftovers"
+  ${CMD} delete apiservice v1alpha3.subresources.kubevirt.io || true
```

Comment:
It is not important but `--ignore-not-found=true` is a useful flag for this kind of stuff. FYI : )